packages/cloud_firestore_odm/cloud_firestore_odm_generator/cloud_firestore_odm_generator_integration_test/test/collection_test.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:cloud_firestore_odm_generator_integration_test/simple.dart';
+import 'package:firebase_core/firebase_core.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'setup_firestore_mock.dart';
+
+void main() {
```

Comment:
I took the liberty to add tests for a bug-fix that was merged without test before